docs/content/operations/druid-console.md

```diff
@@ -86,5 +86,8 @@ The data servers tab lets you see the current status of the historical nodes and
 
 The SQL view lets you issue direct DruidSQL queries and display the results as a simple table. Note that despite the name this view also allows you to enter native Druid queries in Hjson format.
 
-![sql](./img/09-sql.png)
+![sql](./img/09-sql-1.png)
 
+The SQL view allows you to issue Druid's native query which is Hjson over HTTP. To send Druid native query, you must start your query with { and in the format of JSON.
```

Comment:
typo `Hjson`?

Reply:
yup, Qi you are confused, the format is JSON over HTTP (Druid does not parse hjson) but the console does, but do not mention it, it is a surprise.